apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/style.scss

```diff
@@ -3,3 +3,14 @@
 		display: none;
 	}
 }
+
+.post-type-page, .post-type-wp_template_part {
+	.editor-styles-wrapper {
+		padding: 0;
+		margin: 0;
+	}
+
+	.template-block {
+		margin-top: -28px;
```

Comment:
Since we can't target parent selectors accurately, the only way I can see to remove the editor UI spacing is to use a negative margin here. The 28px space is there for editor block insertion UI (the (+) button before blocks) which is never present for these template blocks.

Reply:
This does also make the footer slightly overlap the the blocks above it when I apply this locally:

<img width="663" alt="footer" src="https://user-images.githubusercontent.com/3629020/62344772-7e6cb300-b543-11e9-9080-d598aca4062b.png">

... just looking to see if there is a way to easily target the header template separately

Reply:
Ah yes, I think we can actually remove this negative margin hack by adding top level css classes like here: https://github.com/Automattic/wp-calypso/pull/35081/files#diff-d4f340da4f72898d36eb7f2ca6c1e127R31

Reply:
Wild idea, why not applying the negative margin too to the wrapper?

```css
.editor-styles-wrapper {
	margin-top: -28px;
	padding: 0;
}
```

(AFAICS the wrapper doesn't really need `margin: 0`, but maybe I'm not testing some cases 🤔 )